### PR TITLE
Confirmation state cleanup

### PR DESCRIPTION
**Pull request by jordan-brough**
We've already updated so that the confirm is always included.

I noticed this while looking at some other stuff.

### CODE REVIEW

**Comment by magnusvk** [comment]
This looks good, though the spec failures appear real.

File: guides/content/developer/core/orders.md [hunk 1]
```diff
@@ -63,12 +63,7 @@ The default states are as follows:
 
 The `payment` state will only be triggered if `payment_required?` returns `true`.
 
-The `confirm` state will only be triggered if `confirmation_required?` returns `true`.
```
Comment (athal7) [inline guides/content/developer/core/orders.md]: TIL we haven't yet removed the guides.

Comment (jhawthorn) [inline guides/content/developer/core/orders.md]: We should remove the guides

**Comment by jordan-brough** [comment]
@magnusvk specs are passing now.

File: guides/content/developer/core/orders.md (hunk 1 above)
Comment (jordan-brough) [inline guides/content/developer/core/orders.md]: https://github.com/solidusio/solidus/issues/276

**Comment by magnusvk** [comment]
:+1:

**Comment by jhawthorn** [comment]
:+1:

**Pull request merged**